packages/braid-design-system/color-mode/query-param.ts

```diff
@@ -0,0 +1,59 @@
+import { darkMode } from '../lib/css/atoms/sprinkles.css';
+
+const flag = '_bdsdm';
+// VALUES
+// 0 = light mode
+// 1 = dark mode
+// 2 = OS Preference
+declare global {
+  interface Window {
+    [flag]: '0' | '1' | '2';
+  }
+}
+
+/**
+ * Script to resolve the color mode preference from the query string, used for passing the native mobile app color mode preference to an embedded Braid web UI.
+ *
+ * Pre-minified script tag, designed to be inserted at the beginning of the document head. This ensures the color mode preference is evaluated and applied before any renderering occurs, preventing the flash of light mode.
+ *
+ * ---
+ *
+ * Example usage:
+ *
+ * &nbsp;&nbsp;&lt;html&gt;
+ *
+ * &nbsp;&nbsp;&nbsp;&nbsp;&lt;head&gt;
+ *
+ * &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;!-- Place script here --&gt;
```

Comment:
```suggestion
 * Example usage:
 *
 * ```html
 * <html>
 *  <head>
 *    <!-- Place script here -->
 * ```
```

<img width="407" alt="with Markdown" src="https://user-images.githubusercontent.com/3297808/214482353-48cd4f38-db63-476b-8db0-74abfc5afadb.png">

Reply:
This is beautiful! 😘👌